docs/contributing/contributor-guidelines.md

```diff
@@ -1,3 +1,31 @@
 # <img class="dcr-icon" src="/img/dcr-icons/Question.svg" /> Contributor Guidelines
 
 ---
+
+Decred development is, like the project, decentralized. Different projects and their repos can have their own own rules and norms. However, the general process for contributing across repos will be the same. Below we describe the basic process for finding work, engaging projects and the general GitHub process.
+
+## Finding Work
+
+Nearly all work occurs in public repos and is open to new contributors. A full list of repos can be found in the Decred [GitHub org](https://github.com/decred). A list of projects under active development and descriptions of them can be found on the [Projects](../../projects) page. 
```

Comment:
Let's spells out "organisation" rather than "org". It may not be obvious for those new to github.

Also the link needs to be to the `.md` file - I'm sure one day you will learn :laughing:

Reply:
Done.

>Also the link needs to be to the .md file - I'm sure one day you will learn 😆

It's a bad habbit. I can quit anytime :man_facepalming: